build/config.props

```diff
@@ -19,7 +19,7 @@
 
     <!-- ** Change for each new preview/rc -->
     <!-- Check the VS schedule and manually enter a preview number here that makes sense. -->
-    <ReleaseLabel Condition=" '$(ReleaseLabel)' == '' ">preview.1</ReleaseLabel>
+    <ReleaseLabel Condition=" '$(ReleaseLabel)' == '' ">preview.2</ReleaseLabel>
```

Comment:
I think we should bump to P3 already. 
Main will move to preview.3 on Fri and dev will continue inserting into main.

Reply:
Yes, I noticed that. We're already 1 month late for P2, let's bump to P3.